madmom/ml/nn/layers.py

```diff
@@ -568,6 +568,16 @@ def __getstate__(self):
         return state
 
     def __setstate__(self, state):
+        # TODO: old models have a 'hid_init' instead of an 'init' attribute
+        #       remove this unpickling code after updating all models
```

Comment:
I think we should track such ToDos in the issue tracker. ToDo items in code tend to be forgotten after some time... :smile:

Reply:
I am not sure if all of these TODOs should go into the issue tracker, since there are plenty and some are simply not too urgent. However I still mark them with TODO, since then at least Pylint lists them and they won't be forgotten totally.